README.md

```diff
@@ -6,3 +6,14 @@ This library only supports Intel Processor Trace, but in the future we hope to
 support alternatives such as Arm's CoreSight.
 
 **This is experimental code.**
+
+## Building
+
+Building is performed by Cargo. In short, most users will want:
```

Comment:
I think we can assume that everyone knows that this needs to be built with cargo, so we can remove this sentence and the example.